src/utils/CookieExtractor.ts

```diff
@@ -0,0 +1,16 @@
+export default class CookieExtractor {
+
+    constructor(private readonly cookiesString: string) { }
+
+    public getCookie(name: string) {
+        const cookies = this.cookiesString.split('; ');
+        const result = {};
+        for (const cookie of cookies) {
+            const current = cookie.split('=');
+            result[current[0]] = current[1];
```

Comment:
Check the `current` length and throw an appropriate error if it isn't equal to 2.